gradle.properties

```diff
@@ -11,7 +11,7 @@ gradleVersion=8.5
 
 # Plugin Dependencies -> https://plugins.jetbrains.com/docs/intellij/plugin-dependencies.html
 platformBundledPlugins=com.intellij.java, org.jetbrains.plugins.yaml, org.jetbrains.plugins.terminal
-platformPlugins=com.redhat.devtools.intellij.telemetry:1.2.1.62, com.redhat.devtools.intellij.kubernetes:1.4.0.4
+platformPlugins=com.redhat.devtools.intellij.telemetry:1.2.1.62
```

Comment:
there's no dependency of openshift to kubernetes. The only thing that we may want is that people would install kubernetes plugin when installing the openshift plugin. We need to remove the kubernetes plugin 1.4 though for things to compile. Kubernetes 1.4 is using kubernetes-client 6.x and overrides the direct dependency onto 7.x causing compilation to fail.